config.m4

```diff
@@ -1,11 +1,5 @@
 dnl config.m4 for the solr extension
 
-dnl Configuring the CURL external library
-dnl This folder is the grand-parent folder of easy.h
-PHP_ARG_WITH(curl, for cURL support, [  --with-curl[=DIR]		SOLR : libcurl install prefix])
```

Comment:
I think this can be dropped.
- without is not supported
- giving directory is not supported anymore, users have to set `PKG_CONFIG_PATH` instead (exactly like `PKG_CHECK_MODULES`)

Reply:
true

Reply:
I think it was changed on php-src, let me check

Reply:
no, Its still useful: 
from php-src PHP-8.2
```
./configure --help | grep curl
  --with-curl             Include cURL support
              value of supported_features for libcurl, overriding pkg-config

```

Reply:
It's an option to override pkg-config it seems, I'm no expert on the matter, can u elaborate ?

Reply:
This option in PHP is to enable the curl extension, not to give the library installation directory (as the comment about pkg-config states)